docs/get-started/go.md

```diff
@@ -0,0 +1,176 @@
+# Go Quickstart for ADK
+
+This guide shows you how to get up and running with Agent Development Kit
+for Go. Before you start, make sure you have the following installed:
+
+*   Go 1.24.4 or later
+
+## Create an agent project
+
+Create an agent project with the following files and directory structure:
+
+```none
+my_agent/
+    agent.go    # main agent code
+    go.mod      # module configuration
+    .env        # API keys or project IDs
+```
+
+??? tip "Create this project structure using the command line"
+
+    === "Windows"
+
+        ```console
+        mkdir my_agent\
+        type nul > my_agent\agent.go
+        type nul > my_agent\env.bat
+        ```
+
+    === "MacOS / Linux"
+
+        ```bash
+        mkdir -p my_agent/ && \
+            touch my_agent/agent.go \
+            touch my_agent/.env
```

Comment:
This gets interpreted as:

```
touch my_agent/agent.go touch my_agent/.env
```

and creates a file called `touch`.

You can either add a `&&` before the `\` on the second line, or combine this into a single `touch` command to create both files.

Note that this issue also shows up in the Java page.